docs/current_docs/integrations/snippets/php/go/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+	"fmt"
+)
+
+type MyModule struct{}
+
+// return container image with application source code and dependencies
+func (m *MyModule) Build(source *Directory) *Container {
```

Comment:
Since every function has the `source *Directory` argument, it's simpler to pass it in the constructor. That's what we're doing in our CI.

Reply:
I would have but in a previous discussion we decided not to: https://discord.com/channels/707636530424053791/1120503349599543376/1209223864417124352